### PR TITLE
docs: profile readme change

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Small editorial update to the readme to match changes [in COS](https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/401).

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version. 

Your notes helps the merger write the commit message for the PR that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: profiles/fscloud/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The default values in this profile were scanned by [IBM Code Risk Analyzer (CRA)
 
 > rule-beb7b289-706b-4dc0-b01d-b1d15d4331e3: Check whether Databases for PostgreSQL network access is restricted to a specific IP range.
 
-The IBM Cloud Framework for Financial Services mandates the application of an inbound network-based allowlist in front of the ICD PostgreSQL instance. You can comply with this requirement by using the `cbr_rules` variable in the module, which can be used to create a narrow context-based restriction rule that is scoped to the ICD PostgreSQL instance. Currently CRA does not support checking for context-based restrictions so hence the above failing rule can be ignored once you are setting context-based restrictions.
+The IBM Cloud Framework for Financial Services mandates the application of an inbound network-based allowlist in front of the ICD PostgreSQL instance. You can comply with this requirement by using the `cbr_rules` variable in the module, which can be used to create a narrow context-based restriction rule that is scoped to the IBM Cloud Databases for PostgreSQL instance. CRA does not support checking for context-based restrictions, so you can ignore the failing rule after you are set the context-based restrictions.
```
Comment (ocofaigh) [inline profiles/fscloud/README.md:9]: > after you are set the context-based restrictions.

This doesn't make sense? It is up to the consumer to ensure they set the appropriate CBR rule here. So the way I was wording it was as long as you are setting the CBR rule, you can ignore the failing CRA rule.

Btw, CRA is adding support for CBR checking, so hence why I said "Currently CRA does not support". Not sure if we want to mention that or not?

Comment (SirSpidey) [reply profiles/fscloud/README.md:9]: fixed typo 
_after you ~~are~~ set_

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@SirSpidey Could you resolve conflicts so I can approve and merge? Thanks

**Comment by SirSpidey** [comment]
Yes, it got fixed in #188. Who knew?

**Pull request closed without merging**